src/main/java/github/scarsz/discordsrv/objects/threads/PresenceUpdater.java

```diff
@@ -29,18 +29,21 @@
 import github.scarsz.discordsrv.util.MessageUtil;
 import github.scarsz.discordsrv.util.PlaceholderUtil;
 import github.scarsz.discordsrv.util.PlayerUtil;
+import net.dv8tion.jda.api.OnlineStatus;
 import net.dv8tion.jda.api.entities.Activity;
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.LinkedList;
 import java.util.List;
+import java.util.Locale;
 import java.util.Objects;
 import java.util.concurrent.TimeUnit;
 
 public class PresenceUpdater extends Thread {
 
     private int lastStatusIndex = 0;
     private String lastStatus = null;
+    private String lastOnlineStatus = null;
```

Comment:
Why is this a String? The lastStatus var is a String because it's a message. This is an enum, it should just be stored as the enum.